crates/bdk/src/descriptor/mod.rs

```diff
@@ -545,7 +563,7 @@ impl DescriptorMeta for ExtendedDescriptor {
             return None;
         }
 
-        let descriptor = self.at_derivation_index(0);
+        let descriptor = self.at_derivation_index(0).expect("TODO");
```

Comment:
In d12242ec746ffc51da64672d7c7f7e014258d190:

You can just unwrap. This function only fails if the index is out of range, and 0 is not.

Reply:
Ditto at several other places.